openfisca_us/variables/gov/states/md/tax/income/exemption/md_aged_blind_exemptions.py

```diff
@@ -0,0 +1,58 @@
+from openfisca_us.model_api import *
+
+
+class md_aged_blind_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD aged blind exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://govt.westlaw.com/mdc/Document/NF59A76006EA511E8ABBEE50DE853DFF4?viewType=FullText&originationContext=documenttoc&transitionType=CategoryPageItem&contextData=(sc.Default)"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
```

Comment:
`blind_spouse` is zero for people without spouses in the filing unit, so you can remove this.